### PR TITLE
quic docs: update ddd README.md

### PR DESCRIPTION
**Pull request by jamuir**
Minor edits (expand MVP acronym, suggest how to install "libuv")

These changes are part of
https://github.com/openssl/project/issues/253

##### Checklist

- [x] documentation is added or updated

### CODE REVIEW

**Comment by jamuir** [comment]
Some of my changes are stylistic.  Please push back on those if they don't seem to improve anything.

**Review by paulidale — APPROVED** [review]

File: doc/designs/ddd/README.md [hunk 1]
```diff
@@ -86,8 +86,8 @@ the applications:
 | pgbouncer        | A |      AOSF, BIOc  |
 
 * Blk: Whether the application uses blocking or non-blocking I/O.
-  * S: Blocking
-  * A: Nonblocking
+  * S: Blocking (Synchronous)
+  * A: Nonblocking (Asynchronous)
```
Comment (t8m) [inline doc/designs/ddd/README.md:90]: This is not a synonym and I think it might be confusing to some.

**Review by t8m — COMMENTED** [review]
Except for one thing LGTM.

File: doc/designs/ddd/README.md (hunk 1 above)
Comment (jamuir) [reply doc/designs/ddd/README.md:90]: okay.  Does anyone have a suggestion on what to do here?

Comment (hlandau) [reply doc/designs/ddd/README.md:90]: I don't really mind this terminology.

**Review by hlandau — APPROVED** [review]

**Comment by hlandau** [comment]
Merged to master and 3.2. Thank you.

**Pull request closed without merging**